draft-packages/form/KaizenDraft/Form/TextField/TextField.spec.tsx

```diff
@@ -91,4 +91,35 @@ describe("<TextField />", () => {
 
     expect(input).toBeInTheDocument()
   })
+
```

Comment:
Not sure if it's covered by the test outside of your change, but to be really thorough we should have a test for each of the 6 scenarios:
- type/inputType * text/email/password
- plus 1 that type supersedes inputType
Should be quick and easy to set up.

Reply:
> type/inputType * text/email/password

I think the amount of tests here is a bit overkill, just testing that the `type` is passed correctly should cover all the scenarios...?

Reply:
I mean I've done more overkill in my past 😬 I understand what you mean, and it's probably enough confidence. But copy/pasting what is essentially a 2 line test 6 times and changing the value is ok-ish, too, in my book.

Just to be clear: if you @gyfchong or @mcwinter07 think you have the confidence you need, that is definitely all good with me!

Reply:
This is true, I suppose that two test technically cover the situation we need. One to verify that `type` passes the input type into the `TextField`, the other to verify that `type` will supersede the deprecated `inputType`.

Reply:
I'll just make sure that there are password stories for the Textfield to feed to chromatic to get the visual coverage

Reply:
Updated and ready to roll 👍